wp_mobile_cache/src/lib.rs

```diff
@@ -320,4 +324,28 @@ mod tests {
             MIGRATION_QUERIES.len()
         );
     }
+
+    #[test]
+    fn test_bundled_sqlite_version_supports_returning() {
```

Comment:
I wonder if we can or should expose a function like this within the library layer, then expose it via FFI so that we can run the same check on iOS (where we use the system sqlite)